docs/guides/setup.md

```diff
@@ -56,36 +56,36 @@ In the previous section, we've created a `config.toml` file in our configuration
 
 # Minimum log level for SignCTRL logs.
 # Must be either DEBUG, INFO, WARN or ERR.
-log_level = ""
```

Comment:
These values are left empty/invalid on purpose. This block merely shows what the initial config.toml looks like on generation. Right below, it says exactly that:

"The initial `config.toml` is merely a template and needs customization."

Reply:
We should set reasonable defaults in the initial config.toml where possible

Reply:
I was a bit concerned setting default values, because I wanted to force people to look into the config and not just take the default one and use that for every node in the set, as that could lead to double-signing, say if we set the rank's default value to 1.

Reply:
Agreed, we shouldn't set rank. But default endpoints/ports/log_levels should be fine.

Reply:
Yes, there are checks in place for too low ranks and ranks above the set size limit.

Alright, I'm setting `rank = 0` by default, so it throws an error on initial use.

Reply:
Perfect